waiting_room.go

```diff
@@ -76,6 +76,18 @@ type WaitingRoomRule struct {
 	Enabled     *bool      `json:"enabled"`
 }
 
+// WaitingRoomSettings describes zone-level waiting room settings.
+type WaitingRoomSettings struct {
+	// Whether to allow verified search engine crawlers to bypass all waiting rooms on this zone
+	SearchEngineCrawlerBypass bool `json:"search_engine_crawler_bypass"`
```

Comment:
at this point, i recommend all booleans be defined as pointers due to the number of value distinction issues we've hit in the past.

Reply:
`search_engine_crawler_bypass` will always be present in the response, and will always be either `true` or `false`. I think it would be misleading to make it be a tristate.

Reply:
if it's always present, 👍